submissions/bmukha/hooli-style-popup/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 24px;
+  color: gray;
+}
+
+.header {
+  width: 90vw;
+  min-width: 400px;
+  height: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #dea47e;
+  box-shadow: 1px 5px 15px #000;
+  border-radius: 5px;
+  margin: 5px auto;
+  padding: 20px;
+}
+
+.header * {
+  display: block;
+}
+
+.header__nav {
+  width: 100%;
+  height: 100%;
+}
+.header__nav__items {
+  list-style-type: none;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+.header__nav__item__main:hover,
+.header__nav__item__secondary:hover,
+.more__label:hover {
+  box-shadow: 0 0 10px black;
+  border-radius: 5px;
+}
+.header__nav__item__main,
+.header__nav__item__secondary,
+.more__label {
+  width: auto;
+  margin: 10px;
+  padding: 5px;
+}
+.menu__toggle,
+.more__checkbox {
+  appearance: none;
+}
+.menu {
+  position: relative;
+}
+.menu__wrapper {
+  display: none;
+  position: absolute;
+  top: 75px;
+  right: -30px;
+  background-color: #dea47e;
+  box-shadow: 1px 5px 15px #000;
+  border-radius: 5px;
+}
+
+.menu__toggle:checked ~ .menu__wrapper {
+  display: block;
+  overflow-y: scroll;
+}
+
+.menu__wrapper__secondary {
+  display: none;
+}
+
+.more__checkbox:checked ~ .menu__wrapper__secondary {
+  display: grid;
+}
+
+input[type="checkbox"]:checked + label::after {
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)